website/docs/r/org_vdc.html.markdown

```diff
@@ -0,0 +1,115 @@
+    cpu {
+      units     = "MHz"
+      allocated = 2048
+      limit     = 2048
+      reserved  = 2048
+      used      = 0
+      overhead  = 0
+    }
+
+    memory {
+      units     = "MB"
+      allocated = 2048
+      limit     = 2048
+      reserved  = 2048
+      used      = 0
+      overhead  = 0
+    }
+  }
+
+  storage_profile {
+    name     = "storage-name"
+    enabled  = true
+    limit    = 10240
+    default  = true    
+  }
+
+  enabled                  = true
+  enable_thin_provisioning = true
+  enable_fast_provisioning = true
+  delete_force             = true
+  delete_recursive         = true
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `org` - (Optional) Organization to create the VDC in, optional if defined at provider level
+* `name` - (Required) VDC name
+* `description` - (Optional) VDC friendly description
+* `provider_vdc_name` - (Required) A name of the Provider VDC from which this organization VDC is provisioned.
+* `allocation_model` - (Required) The allocation model used by this VDC; must be one of {AllocationVApp, AllocationPool, ReservationPool}
+* `compute_capacity` - (Required) The compute capacity allocated to this VDC.  See [Compute Capacity](#computecapacity) below for details.
+* `nic_quota` - (Optional) Maximum number of virtual NICs allowed in this VDC. Defaults to 0, which specifies an unlimited number.
+* `network_quota` - (Optional) Maximum number of network objects that can be deployed in this VDC. Defaults to 0, which means no networks can be deployed.
+* `vm_quota` - (Optional) The maximum number of VMs that can be created in this VDC. Includes deployed and undeployed VMs in vApps and vApp templates. Defaults to 0, which specifies an unlimited number.
+* `enabled` - (Optional) True if this VDC is enabled for use by the organization VDCs. Default is true.
+* `storage_profile` - (Required) Storage profiles supported by this VDC.  See [Storage Profile](#storageprofile) below for details.
+* `memory_guaranteed` - (Optional) Percentage of allocated memory resources guaranteed to vApps deployed in this VDC. For example, if this value is 0.75, then 75% of allocated resources are guaranteed. Required when AllocationModel is AllocationVApp or AllocationPool. Value defaults to 1.0 if the element is empty.
+* `cpu_guaranteed` - (Optional) Percentage of allocated CPU resources guaranteed to vApps deployed in this VDC. For example, if this value is 0.75, then 75% of allocated resources are guaranteed. Required when AllocationModel is AllocationVApp or AllocationPool. Value defaults to 1.0 if the element is empty.
+* `cpu_frequency` - (Optional) Specifies the clock frequency, in Megahertz, for any virtual CPU that is allocated to a VM. A VM with 2 vCPUs will consume twice as much of this value. Ignored for ReservationPool. Required when AllocationModel is AllocationVApp or AllocationPool, and may not be less than 256 MHz. Defaults to 1000 MHz if value isn't provided.
```

Comment:
GUI would be matched with `cpu_speed`. What would be your and @Didainius and @dataclouder vote?

Reply:
cpu_speed sounds good

Reply:
changed